cmd/res/profiles/llrp.impinj.profile.yaml

```
@@ -0,0 +1,154 @@
+    properties:
+      valueType: "Object"
+      readWrite: "R"
+
+  - 
+    name: "ReaderConfig"
+    description: "A Reader's configuration"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "ROSpecID"
+    description: "Client-generated Reader Operation Specification Identifier"
+    properties:
+      valueType: "uint32"
+      readWrite: "W"
+      defaultValue: "1"
+
+  - 
+    name: "AccessSpecID"
+    description: "Client-generated Access Specification Identifier"
+    properties:
+      valueType: "uint32"
+      readWrite: "W"
+      defaultValue: "1"
+
+  - 
+    name: "ROSpec"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "AccessSpec"
+    properties:
+      valueType: "Object"
+      readWrite: "RW"
+
+  - 
+    name: "Action"
+    properties:
+      valueType: "String"
+      readWrite: "W"
+
+  - 
+    name: "ROAccessReport"
+    isHidden: true # resource is only async
+    properties:
+      valueType: "Object"
+      readWrite: "R" 
+
+  - 
+    name: "ReaderEventNotification"
+    isHidden: true # resource is only async
+    properties:
+      valueType: "Object"
+      readWrite: "R" 
+
+deviceCommands:
+  - 
+    name: enableROSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "ROSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Enable" }
+
+  - 
+    name: startROSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "ROSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Start" }
+
+  - 
+    name: stopROSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "ROSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Stop" }
+
+  - 
+    name: disableROSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "ROSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Disable" }
+
+  - 
+    name: deleteROSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "ROSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Delete" }
+
+  - 
+    name: enableAccessSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "AccessSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Enable" }
+
+  - 
+    name: disableAccessSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "AccessSpecID", defaultValue: "0" }
+      - { deviceResource: "Action", defaultValue: "Disable" }
+
+  - 
+    name: deleteAccessSpec
+    isHidden: false
+    readWrite: "W"
+    resourceOperations:
+      - { deviceResource: "AccessSpecID" }
+      - { deviceResource: "Action", defaultValue: "Delete" }
```

Looks like when you deleted the extra whitespace on the last line, you removed the newline itself.
```suggestion
      - { deviceResource: "Action", defaultValue: "Delete" }

```

---

re-added the new line